.github/workflows/release.yaml

```diff
@@ -91,7 +91,8 @@ jobs:
         with:
           app_id: ${{env.APP_ID}}
           private_key: ${{ secrets.TOKEN_EXCHANGE_GH_APP_PRIVATE_KEY }}
-          repositories: [ ${{ github.repository }} ]
+          repositories:
+            - ${{ github.repository }} ]
```

Comment:
Do we still need the `]`?

Reply:
No definitely not, moved through this a bit too quickly! ^^"